haskeline.cabal

```diff
@@ -39,7 +39,8 @@ flag terminfo
     Default: True
     Manual: True
 
-Library
+Library haskeline-lib
```

Comment:
It looks like this line broke the Travis CI.  Older versions of Cabal don't like giving the library component a name:
https://travis-ci.org/judah/haskeline/builds/271286596?utm_source=github_status&utm_medium=notification